scripts/deploy/03_wbtc_call_theta_vault.ts

```diff
@@ -56,20 +56,8 @@ const main = async ({
   });
 
   const logicDeployment = await deployments.get("RibbonThetaVaultLogic");
-  const lifecycle = await deployments.get("VaultLifecycle");
-
-  // Supports Uniswap V3 only
-  const dexRouter = await deploy("UniswapRouter", {
-    contract: "UniswapRouter",
-    from: deployer,
-  });
 
-  const RibbonThetaVault = await ethers.getContractFactory("RibbonThetaVault", {
-    libraries: {
-      VaultLifecycle: lifecycle.address,
-      UniswapRouter: dexRouter.address, // Supports only Uniswap v3
```

Comment:
shouldn't we link the libraries

Reply:
It was failing because they were linked. I will give this a second look after we get the linting MR merged.